### PR TITLE
iOS base UI and functions

### PR DESCRIPTION
**Pull request by Amanida**
Added base UI : TextView for input text, 'Play' Button, 'Select Language Button. Title bar
Completed feature : Playing text with Korean voice
Now working : Adding action sheet with picker View. Selecting language feature.

### CODE REVIEW

**Comment by youknowone** [comment]
I found the width of the alert view becomes shorter after presenting it.
Insert the following code:

```
        self.languagePicker.autoresizingMask = [UIViewAutoresizing.flexibleLeftMargin, UIViewAutoresizing.flexibleRightMargin]
```

**Comment by Amanida** [comment]
Thank you for review. I applied your code and resizing picker view. Please check this tomorrow...not today! :)

File: iOS/Speaker.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+//
+
+import Foundation
+import AVFoundation
+
+public class Speaker: NSObject, AVSpeechSynthesizerDelegate {
+    static let defaultSpeaker = Speaker()
+    var volume: Float = 1.0
+    var mute = false
+    var voice = AVSpeechSynthesisVoice(language: "ko-KR")     // set default language : korean
```
Comment (youknowone) [inline iOS/Speaker.swift]: Say shouldn't set korean as default language. We should guess system default language, but "en-US" will be ok for the first step.

File: iOS/Speaker.swift [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    public func speechSynthesizer(_ synthesizer: AVSpeechSynthesizer, didFinish utterance: AVSpeechUtterance) {
+//        Do what speeching finished
+        print("speeching finished")
+    }
+
+    public func speakText(text: String) {
+        let utterance = AVSpeechUtterance(string: text)
+        utterance.rate = 0.4
```
Comment (youknowone) [inline iOS/Speaker.swift]: Default speech rate will be better.

File: iOS/ViewController.swift [hunk 3]
```diff
@@ -8,18 +8,70 @@
...
+    var languageNames = [String]()
+    
+    @IBAction func playClicked(_ sender: AnyObject) {
+        if let text = self.textview.text {
+            self.speaker.speakText(text: text)
+        }
+    }
+    @IBAction func changeVoiceClicked(_ sender: AnyObject) {
+        // create action sheet dynamically
+        let alertView = UIAlertController(title: "Select Launguage", message: "\n\n\n\n\n\n\n\n\n\n", preferredStyle: UIAlertControllerStyle.actionSheet);
```
Comment (youknowone) [inline iOS/ViewController.swift]: `alertController` instead of `alertView`

File: iOS/ViewController.swift [hunk 4]
```diff
@@ -8,18 +8,70 @@
...
+    func speaker(speaker: Speaker, didFinishSpeechString: String) {
+        print("didFinishSpeechString : ", didFinishSpeechString)
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+        
+        // create popup picker view
+        languagePicker = UIPickerView(frame: CGRect(x: 0, y: 20, width: 300, height: 250))
```
Comment (youknowone) [inline iOS/ViewController.swift:45]: I think the size must not be set as fixed, but flexible ralative to the alert view width.
for example:
width = alertView's width - 40

**Review by youknowone — CHANGES_REQUESTED** [review]

**Pull request merged**